src/olympia/addons/serializers.py

```diff
@@ -210,8 +210,21 @@ class Meta:
 
     def to_representation(self, obj):
         data = super(AddonSerializer, self).to_representation(obj)
-        if data['theme_data'] is None:
+        if 'theme_data' in self.fields and data['theme_data'] is None:
             data.pop('theme_data')
+        if 'homepage' in self.fields:
+            data['homepage'] = self.outgoingify(data['homepage'])
```

Comment:
That should not happen, no, but maybe I should just do `in data` instead of `in self.fields`, I'll look into that.